bridges/chains/chain-asset-hub-westend/src/lib.rs

```diff
@@ -117,3 +117,34 @@ impl Parachain for AssetHubWestend {
 	const PARACHAIN_ID: u32 = ASSET_HUB_WESTEND_PARACHAIN_ID;
 	const MAX_HEADER_SIZE: u32 = MAX_ASSET_HUB_HEADER_SIZE;
 }
+
+/// Describing permissionless lanes instance
+impl ChainWithMessages for AssetHubWestend {
+	const WITH_CHAIN_MESSAGES_PALLET_NAME: &'static str =
+		WITH_ASSET_HUB_WESTEND_MESSAGES_PALLET_NAME;
+
+	const MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX: MessageNonce =
+		MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX;
+	const MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX: MessageNonce =
+		MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX;
+}
+
+/// Public key of the chain account that may be used to verify signatures.
+pub type AccountSigner = MultiSigner;
+
+/// The address format for describing accounts.
+pub type Address = MultiAddress<AccountId, ()>;
+
+/// Name of the With-AssetHubWestend messages pallet instance that is deployed at bridged chains.
+pub const WITH_ASSET_HUB_WESTEND_MESSAGES_PALLET_NAME: &str = "BridgeWestendMessages";
+
+/// Name of the With-AssetHubWestend bridge-relayers pallet instance that is deployed at bridged
+/// chains.
+pub const WITH_ASSET_HUB_WESTEND_RELAYERS_PALLET_NAME: &str = "BridgeRelayers";
+
+/// Pallet index of `BridgeRococoMessages: pallet_bridge_messages::<Instance1>`.
+pub const WITH_BRIDGE_WESTEND_TO_ROCOCO_MESSAGES_PALLET_INDEX: u8 = 60; // TODO: FAIL-CI - correct index when AssetHubWestend
```

Comment:
@serban300 thank you, good catch: https://github.com/paritytech/polkadot-sdk/pull/6675/files#diff-a81a7d2eaeeb3b6f1256b7b96b1f43a2447943f498f053ecf1e511bdbfa04cd1R1199

```suggestion
pub const WITH_BRIDGE_WESTEND_TO_ROCOCO_MESSAGES_PALLET_INDEX: u8 = 63;
```